.github/ISSUE_TEMPLATE.md

```diff
@@ -2,9 +2,9 @@
 
 This issue tracker is only for technical issues related to garlicoin-core.
 
-General garlicoin questions and/or support requests and are best directed to the [garlicointalk.io forums](https://garlicointalk.io/).
+General garlicoin questions and/or support requests and are best directed to the [garlicoin subreddit](https://www.reddit.com/r/garlicoin/) or [garlicoin discord server](https://discord.gg/garlicoin).
 
-For reporting security issues, please contact the Garlicoin developers on the #garlicoin-dev Freenode IRC channel or alternatively you can email us at contact@garlicoin.org.
+For reporting security issues, please contact the Garlicoin developers on the #garlicoin-dev Freenode IRC channel or alternatively you can email us at help@garlicoin.com.
```

Comment:
We don't currently have a freenode channel nor do we own the garlicoin.com domain.

Reply:
yeah, this was just left in from search/replace. whole section needs to be removed